samples/dashboards/sync/custom-component-sync/demo.js

```diff
@@ -0,0 +1,369 @@
+const { Component, ComponentRegistry } = Dashboards;
+
+class Slider extends Component {
+    constructor(cell, options) {
+        super(cell, options);
+        this.type = 'Slider';
+        this.createDOMStructure();
+        this.xColumn = [];
+        this.sync = new Component.Sync(
+            this,
+            this.syncHandlers
+        );
+        return this;
+    }
+
+    createDOMStructure() {
+        this.slider = document.createElement('input');
+        this.handleLabel = document.createElement('span');
+        this.maxSliderLabel = document.createElement('span');
+        this.minSliderLabel = document.createElement('span');
+        this.middleSliderLabel = document.createElement('span');
+
+        const leftSliderSide = document.createElement('div');
+        const rightSliderSide = document.createElement('div');
+
+        this.handleLabel.className = 'sc-handle-label';
+        this.slider.className = 'sc-slider';
+        leftSliderSide.className = 'sc-slider-side';
```

Comment:
I think that you can optimize it a bit.

You can use `createElement` from the HC.

```
this.leftSliderSide = createElement(
            'div', {
                className: 'sc-slider-side'
            }, {},
            this.minSliderLabel
        );
```